config/webpack/paths.yml

```diff
@@ -0,0 +1,18 @@
+default: &default
+  output: public
+  assets: webpack
+  manifest: manifest.json
+  node_modules: client/node_modules
+  source: client/app
```

Comment:
Probably not necessary. See https://github.com/shakacode/react_on_rails/pull/834